### DISCUSSION TITLE
[Bar chart] multiple group by categories

### DISCUSSION THREAD

**Discussion by graceguo-supercat** [apache/superset#18480] [Ideas]
**Is your feature request related to a problem? Please describe.**
User can create bar chart with multiple group by categories. For example, i want to group by `severity` (4 values: critical, high, medium and low) and `ticket_status`(2 values: ticketed and non-ticketed)
![119540330-2fa89280-bd42-11eb-82ea-2f21431e970c-2-2](https://user-images.githubusercontent.com/27990562/119540641-80b88680-bd42-11eb-9322-6940ffaf8182.jpg)

The result bar chart has 4 x 2 bar for each month.

**Describe the solution you'd like**
Now we want to **_stack the Ticketed and Non-Ticketed_** bars for **_each severity_**. so it shows Critical ticketed vs non-tickets, High ticketed vs non-ticketed, Medium ticketed vs non-ticketed, and Low ticketed vs non-ticketed.

Is it possible for bar chart to allow user choose `Primary` and `Secondary` groupby categories (or by the order that user added into `GROUP BY` control)? So that we can display secondary category inside primary category bar 🦸 📊

**Comment by junlincc** [comment]
Thanks for the requests! @graceguo-supercat we are planning for new Echarts Bar, feel free to o file all requirements in this threat, we will try to get all in.
